crates/evm/core/src/backend/mod.rs

```diff
@@ -1856,9 +1857,9 @@ pub(crate) fn merge_account_data<ExtDB: DatabaseRef>(
 
     // need to mock empty journal entries in case the current checkpoint is higher than the existing
     // journal entries
-    while active_journaled_state.journal.len() > target_fork.journaled_state.journal.len() {
-        target_fork.journaled_state.journal.push(Default::default());
-    }
+    // while active_journaled_state.journal.len() > target_fork.journaled_state.journal.len() {
+    // target_fork.journaled_state.journal.push(Default::default());
+    // }
```

Comment:
Note: Revm 21+ flattens the `journal` field to `Vec<JournalEntry` from `Vec<Vec<JournalEntry>>`

I'm trying to understand whether this has side effects for our multiforking

Do you remember the context of this code block @mattsse?